system/load_average.tf

```diff
@@ -0,0 +1,71 @@
+resource "datadog_monitor" "load_average_1" {
+  count   = "${var.monitor_enabled}"
+  name    = "High load average 1 ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "High load average 1 last ${var.load_average["time"]} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.public_ip}"
+  query   = "avg(last_${var.load_average["time"]}):avg:system.load.1{host:${data.aws_instance.monitored.instance_id}} by {host} > ${var.load_average["critical_threshold"]}"
+
+  thresholds {
+    ok       = "${var.load_average["ok_threshold"]}"
+    warning  = "${var.load_average["warning_threshold"]}"
+    critical = "${var.load_average["critical_threshold"]}"
+  }
+
+  renotify_interval = "${var.renotify_interval_mins}"
```

Comment:
I am torn. It makes sense why you added `_mins` suffix, but we don't do it on the others. I think we should just drop it.